### PR TITLE
mobile: fix mobile interface

### PR DESCRIPTION
**Pull request by rjl493456442**
This PR fixes a couple of mobile interface issues. The major one is for `Interfaces.Get` function.  

In the previous version, the `Interfaces.Get` function returns a new created `Interface` with specifed `object` inside. But in the `Interface.SetXXX` functions, actually the `object` will be recreated with a difference address.

So that it means in the below code snippet, we can't change the `args` value through `arg`. 

```
args := NewInterfaces(1);
arg := args.Get(0); 
arg.SetXXX();
// The args will not be effected at all.
```

### CODE REVIEW

File: mobile/common.go [hunk 1]
```diff
@@ -228,3 +228,17 @@ func (a *Addresses) Set(index int, address *Address) error {
...
+//
+// Deprecated: use hexutil.Encode instead.
+func ToHex(b []byte) string {
+	return common.ToHex(b)
+}
+
+// FromHex returns the bytes represented by the hexadecimal string s.
+// s may be prefixed with "0x".
+func FromHex(s string) []byte {
+	return common.FromHex(s)
```
Comment (fjl) [inline mobile/common.go]: Please change both of these functions to use package hexutil. FromHex should return an error.

**Comment by fjl** [comment]
Maybe better idea would be to make functions `NewBoolInterface`, `NewStringInterface`, etc. and use them in the template like this:

```text
{{range $index, $item := .Normalized.Inputs}}args.set({{$index}}, Geth.new{{namedtype (bindtype .Type) .Type}}Interface({{.Name}})); 
```

**Comment by karalabe** [comment]
I guess

```
args := NewInterfaces(1);
arg := args.Get(0); 
arg.SetXXX();
// The args will not be effected at all.
```

Is working as intended. The current way to achieve what you're trying is:

```
args := NewInterfaces(1);
args.Set(0, NewXXXInterface(...));
```

or 

```
args := NewInterfaces(1);

arg := args.Get(0); 
arg.SetXXX();

args.Set(0, arg);
// The args will not be effected at all.
```

**Comment by rjl493456442** [comment]
@fjl @karalabe Update a bit since the last review. PTAL

**Pull request merged**